lib/index.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const fs = require('fs'),
+  path = require('path');
+
+const prompt = require('prompt'),
+  Bluebird = require('bluebird'),
+  log = require('npmlog'),
+  fetch = require('node-fetch'),
+  Joi = require('joi');
+
+Bluebird.promisifyAll(prompt);
+Bluebird.promisifyAll(Joi);
+Bluebird.promisifyAll(fs);
+
+const raveUri = 'http://flw-pms-dev.eu-west-1.elasticbeanstalk.com/flwv3-pug/getpaidx/api/flwpbf-inline.js';
+const configFile = path.resolve('../config.json');
+const schema = require('../schemas').base;
+
+prompt.start();
+prompt.getAsync(Object.keys(schema))
+  .then((result) => Joi.validateAsync(result, schema))
+  .then((value) => log.info('Success', value))
```

Comment:
Does this resolve with the `value`? So that the next function in the chain doesn't get invoked with `undefined`.